daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -75,6 +75,113 @@ Code | Description
 
 This API does not return any content.
 
+## Raise Event request
```

Comment:
Raise and event or register for this event to be raised?

Reply:
Raise an event

Reply:
Either way I still do not understand how to use this API. Can we add
1) A description. "Raise an event from this workflow"
2) Describe when this is used. If this raises an event, how do you listen to this event and can a workflow also be trigger on the event. Please add a description here on the scenario both for workflow and how an app utilizes the event.

Reply:
I responded to the conversation above and recommended a change to how we describe his API. Hopefully that adds a bit more clarity.

One thing we need to mention is that the exact mechanism for subscribing to an event depends on the workflow component that you're using. Dapr Workflow has one way of subscribing to external events but other workflow components might have different ways.